api/src/main/java/cz/muni/fi/pa165/facade/HallFacade.java

```diff
@@ -0,0 +1,21 @@
+package cz.muni.fi.pa165.facade;
+
+import cz.muni.fi.pa165.dto.HallDTO;
+import java.util.List;
+
+/**
+ *
+ * @author xtrnkal
+ */
+public interface HallFacade {
+
+    public Long createHall(HallDTO newHall);
+
+    public List<HallDTO> getAllHalls();
+
+    public HallDTO getHallById(Long id);
+
+    public void removeHall(HallDTO hall);
```

Comment:
Remove by mělo stačit jen id, ne? Ano, taky jsem tenhle komentář dostal, taky jsem to ještě neudělal, ale tu webovou vrstvu to zjednoduší.

Reply:
upraveno, mazání na základě id